qgis_deployment_toolbelt/jobs/job_qgis_installation_finder.py

```diff
@@ -217,15 +230,28 @@ def _get_qgis_bin_in_install_dir(install_dir: str) -> str | None:
                     return qgis_exe
         return None
 
-    @staticmethod
-    def _get_windows_installed_qgis_path() -> dict[str, str]:
+    def _get_search_paths_with_environment_variable(self) -> list[str]:
+        """Get search_paths option with environment variable update
+
+        Returns:
+            list[str]: search_paths
+        """
+        # search_paths
+        search_paths = []
+        if "search_paths" in self.options:
+            for path in self.options["search_paths"]:
+                search_paths.append(expandvars(path))
```

Comment:
@Guts I'm not sure about the `expandvars`. Should we use ` expandvars(expanduser(getenv(path,path))`

Reply:
`~/dev` search was not working on Linux so I updated to use `expanduser` and `getenv`